README.md

```diff
@@ -30,6 +30,10 @@ If you'd like to help out, please see [CONTRIBUTING.md](CONTRIBUTING.md).
 
 ## Release history:
 
+* 0.6.3 (2023-06-14)
+  - Bump MSRV to 1.60.
+  - Skip time invalid certificates.
```

Comment:
Is this meant to describe #45? The description doesn't quite make sense to me, maybe something like: "Avoid storing expired certificates on Windows"?

Reply:
Right, I was wording it like that, poorly I agree, because it's not only expired ones, but future ones as well.
Expires might be enough?

Reply:
"Windows: avoid storing certificates which are currently invalid"?

Reply:
Done.